src/pages/AMM/Swap/components/SwapForm/SwapForm.tsx

```diff
@@ -222,7 +222,6 @@ const SwapForm = ({
   };
 
   const handleTickerChange = (ticker: string, name: string) => {
-    form.setFieldValue(name, ticker, true);
```

Comment:
Is this supposed to be removed?

Reply:
yes, I added some code on the `useForm` that passes the necessary handlers which replaces this